src/meta/meta_table.c

```diff
@@ -203,6 +207,24 @@ __wt_metadata_insert(WT_SESSION_IMPL *session, const char *key, const char *valu
     return (ret);
 }
 
+/*
+ * __wt_metadata_bump_turtle --
```

Comment:
The way `__wti_turtle_update` is written we can only update the metadata uri information. I figured rather than rework the turtle file code we can set the live restore logic as a side effect of touching the turtle file (the implementation is in meta_turtle.c below).
If we want to clean this up I can do so in a follow on ticket.

Reply:
There already exists this exact function with a different name. `meta/meta_table.c:__wt_metadata_turtle_rewrite`.

Reply:
I've fixed this in https://github.com/wiredtiger/wiredtiger/pull/11590/commits/4d23ae032f20f588bec88cbe82ac3d674261768e.
Can you give this commit a careful review? I've added a condition that the turtle lock must be held when calling `__wt_metadata_turtle_rewrite` to protect against concurrent modification which changes assumptions in non-live restore code